### ISSUE TITLE
need android tv support

### ISSUE THREAD

**Issue by zzycami** [SkyD666/AniVu#78]
RT

**Comment by SkyD666** [comment]
tv版逻辑跟手机版有很大不同，sdk也有差异。除非重开一个tv仓库，不然比较困难

**Comment by P0bo** [comment]
im very opposed to the idea of adding tv compatibility to the base app because many apps has tried it and failed . for example adding remote support on aniyomi for tv compatibility raised these issues in latest version 
https://github.com/aniyomiorg/aniyomi/issues/1713

**Comment by P0bo** [comment]
if u can hae a seperate build then it shoud be good to go , but not in the main app

**Comment by onnesoom** [comment]
> im very opposed to the idea of adding tv compatibility to the base app because many apps has tried it and almost always have faced some issues . for example adding remote support on aniyomi for tv compatibility raised these issues in latest version [aniyomiorg/aniyomi#1713](https://github.com/aniyomiorg/aniyomi/issues/1713)

agreed , unless the support for remote is comparable to smth like cloudstream or M3UANDROID its a bad idea to add android tv , even after that , it has many limitations . also since it targets very specific group of users the feedback will be less too , making it slow to evaluate

**Comment by zzycami** [comment]
My tv need it！！！

**Comment by P0bo** [comment]
| [Preview Video](https://github.com/P0bo/aniyomi-extensions/raw/master/buildSrc/Record_2024-08-14-12-47-292.mp4) |
|---|

it seems like for some parts the app works with tv remotes . @SkyD666 perhaps u can try improving  button focusing and clicking with arrows and enter to make the experience better .
